docs/cryptodoc/src/05_09_frodokem.rst

```diff
@@ -0,0 +1,296 @@
+   | :ref:`Constants <pubkey/frodokem/modes>`                       | :srcref:`src/lib/pubkey/frodokem/frodokem_common/frodo_constants.h` | Parameter set instantiations                                                                                                                                                           |
+   +----------------------------------------------------------------+---------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+   | :ref:`Matrix Operations <pubkey/frodokem/matrix_operations>`   | :srcref:`src/lib/pubkey/frodokem/frodokem_common/frodo_matrix.h`    | Matrices and operations on them                                                                                                                                                        |
+   +----------------------------------------------------------------+---------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+   | FrodoKEM                                                       | :srcref:`src/lib/pubkey/frodokem/frodokem_common/frodokem.h`        | FrodoKEM :ref:`Key Generation <pubkey/frodokem/key_generation>`, :ref:`Encapsulation <pubkey/frodokem/encapsulation>`, :ref:`Decapsulation <pubkey/frodokem/decapsulation>`            |
+   +----------------------------------------------------------------+---------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Algorithm Internals
+-------------------
+
+..  _pubkey/frodokem/types:
+
+Types
+^^^^^
+
+For similar reasons as for :ref:`SPHINCS+ strong types <signatures/sphincsplus/types>`,
+Botan's FrodoKEM implementation relies on the use of strong types.
+As most data is just defined as byte sequences, the usage of strong types ensures that
+the correct data is used at each step of the computation.
+More concretely, strong types for the different kinds of seeds, random samples, matrix
+representations, and plaintexts are used in addition to types corresponding to the values
+:math:`salt`, :math:`k`, and :math:`pkh` of [FrodoKEM-ISO]_.
+
+..  _pubkey/frodokem/modes:
+
+Modes and Constants
+^^^^^^^^^^^^^^^^^^^
+
+The implementation realizes all parameter sets shown in Table
+:ref:`Supported FrodoKEM signature algorithms <pubkey/frodokem/parameter_table>`.
```

Comment:
The linked table does not contain any reference to signatures in the title. Should maybe be changed to avoid confusion?

Reply:
Certainly. No idea where the mention of "signature" came from. Copy & Paste oversight, perhaps?